tests/Tests.FeatureManagement/appsettings.json

```diff
@@ -10,7 +10,7 @@
     "OnTestFeature": true,
     "OffTestFeature": false,
     "FeatureUsesFiltersWithDuplicatedAlias": {
-      "RequirementType": "all",
```

Comment:
These should still be lower case

Reply:
Why? It's `All` in the main branch

Reply:
I had this backwards. They should PascalCase.